app/build.gradle

```diff
@@ -69,11 +69,16 @@ detekt {
 
 repositories {
     maven { url "https://jitpack.io" }
+    maven { url 'https://maven.microblink.com' }
 }
 
 dependencies {
     implementation project(":vgscollect")
     implementation project(":vgscollect-cardio")
+    implementation project(":vgscollect-blinkcard")
+    implementation('com.microblink:blinkcard:2.6.0@aar') {
```

Comment:
should we update to 2.7.0?

Reply:
It's demo module, I have updated scanner module, but yes it can be upgradet